### ISSUE TITLE
ex command completion - match on help too?

### ISSUE THREAD

**Issue by bovine3dom** [cmcaine/tridactyl#944]
Probably worth experimenting with this to see if it works OK. Can probably just follow the pattern in buffer completions.

**Comment by glacambre** [comment]
I thought about it but I think it wouldn't work with our current completion mechanism. How can we tell the difference "tabopen I'm looking for something on google", which should obviously provide history completion, and "tabopen is the function I want to know about", which should obviously provide doc completion?

**Comment by bovine3dom** [comment]
Yeah, you're right. We'd need multiple completions at once for this.

**Comment by glacambre** [comment]
Related issue: https://github.com/tridactyl/tridactyl/pull/932

**Comment by bovine3dom** [comment]
We could just have it as a last fallback: if no excmds are recognised, provide completions from documentation.

**Comment by glacambre** [comment]
Related issue: #1175
